Add tooltips to Wiki

CrackTheCode016 opened the pull request:
This PR adds hoverable tooltips for certain definitions/keywords in the Wiki. We need to define which terms to add/remove, but this list is a starting point.

Closes issue #5225

CrackTheCode016 committed:
add initial definitions and tooltips

filippoweb3 commented on includes/definitions.md:
```diff
@@ -0,0 +1,80 @@
+*[Active Nomination]: A validator selected by a nominator to validate and earn rewards.
```
```suggestion
*[Active Nomination]: A validator that produces blocks in the current era and earns staking rewards.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Asset Hub]: A system chain for asset management and more.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Availability Cores]: Entry points to Polkadot's security and interoperability. Time on a core is guaranteed by purchasing coretime.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Blockspace]: Umbrella term to define the resource created and often sold by a blockchain network.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Block Explorer]: A tool for exploring blockchain data through visualization and summaries.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Bonding]: Freezing tokens for staking and other on-chain activities.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Collator]: A node producing parachain blocks.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Collectives]: A parachain hosting on-chain collectives like the Polkadot Technical Fellowship.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Commission]: A validator's rate deducted from rewards to cover operational costs and other expenses.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Common Good (Parachain)]: An old term used to define system chains like the Polkadot Asset Hub.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Coretime]: Time allocated for using a Polkadot's virtual core.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Governance]: The process of stakeholder voting on network changes and other organizational matters.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Inactive Nomination]: A validator that is nominated but not validating and producing blocks in the current era.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Injected Account]: A specific term used to tag an account accessed via the Polkadot-JS UI but managed externally (for example, via a browser extension).
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Interoperability]: The ability of systems to exchange information. In the blockchain context, interoperability enables systems with different consensus to communicate safely.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Kusama]: Polkadot's canary network for early releases and experimentation in general.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Message]: Data sent between parachains using the XCM format.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Parachain]: A layer-one blockchain running on a Polkadot virtual core (in parallel to the relay chain).
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Parity Technologies]: The company that was commissioned to launch Polkadot, and that is currently behind the Polkadot SDK.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Polkadot]: A service provider for secure and resilient computation and blockchain interoperability.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[Web3 Foundation]: A foundation fostering and supporting web3 technologies.
```

filippoweb3 commented on includes/definitions.md:
```suggestion
*[WebAssembly]: A virtual machine instruction format that is used for Polkadot runtimes.
```

filippoweb3 reviewed (changes requested):
lgtm, added some improvements to the definitions. Although we can also improve later on, let's give it a final round of checks.

CrackTheCode016 committed:
Apply suggestions from code review

Co-authored-by: Filippo <110459737+filippoweb3@users.noreply.github.com>

CrackTheCode016 committed:
edits

CrackTheCode016 commented:
I went through and checked each one for inaccuracies, changed a few, should be good now! I also removed any 'generic' ones which didn't need a definition imo.
